airflow/migrations/versions/0050_3_0_0_add_dagbundlemodel.py

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Add DagBundleModel.
+
+Revision ID: e229247a6cb1
+Revises: eed27faa34e3
+Create Date: 2024-12-02 22:06:40.587330
+"""
+
+from __future__ import annotations
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy_utils import UUIDType
+
+from airflow.utils.sqlalchemy import ExtendedJSON, UtcDateTime
+
+revision = "e229247a6cb1"
+down_revision = "eed27faa34e3"
+branch_labels = None
+depends_on = None
+airflow_version = "3.0.0"
+
+
+def upgrade():
+    op.create_table(
+        "dag_bundle",
+        sa.Column("id", UUIDType(binary=False), nullable=False),
+        sa.Column("name", sa.String(length=200), nullable=False),
```

Comment:
We should probably use `airflow.migrations.db_types.StringID` so `collation_args` are applied for free

Reply:
Good call: #44678